src/Storages/MergeTree/SimpleMergeSelector.cpp

```diff
@@ -28,7 +28,7 @@ struct Estimator
         {
             double difference = std::abs(log2(static_cast<double>(sum_size) / size_prev_at_left));
             if (difference < settings.heuristic_to_align_parts_max_absolute_difference_in_powers_of_two)
-                current_score *= interpolateLinear(settings.heuristic_to_align_parts_max_score_adjustment, 1,
+                current_score *= std::lerp(settings.heuristic_to_align_parts_max_score_adjustment, 1,
```

Comment:
`interpolateLinear` was much more readable than `std::lerp` but okay...

Reply:
I believe it's better to use std when possible